exercises/ruby-1-a/README.md

```diff
@@ -4,17 +4,34 @@ This is Part 1 of our research into how people write Ruby differently. This exer
 
 ## Instructions
 
-Your task is to determine which word in a sentence has the most vowels (a, e, i, o, u). A sentence is defined as a [Ruby string][docs-string] consisting of one or more words. A word consists of one or more upper- or lowercase letters. Words are separated from each other by a single space character.
+Your task is to define a shopping cart for a corner shop. The cart should keep track of the list of items added, and correctly calculate the price.
 
-Your job is to define a method with the signature `word_with_most_vowels`, which takes a sentence, and returns the word with the most vowels.
+Items are added to the basket using stock-keeping unit (SKU) codes. The items in the shop are:
+
+| Name      | Price | SKU    |
+|-----------|-------|--------|
+| Potatoes  | 10    | D69856 |
+| Rice      | 30    | F55690 |
+| Cofee     | 14.99 | C662F6 |
+| Newspaper | 2.99  | B48C0D |
```

Comment:
If possible define the SKU with a prefix code that might make sense.  If I want all staples, or all media, perhaps I can gather all available products that start with S or M.  It mirrors internal sku usage (at least for our jewelry company) so that we can compare items to the customer as well "You viewed this, you may be interested in these!" type of an ability.  At the moment, at least to me, the SKU doesn't seem to make sense.

Reply:
I guess at the moment it's only an identifier something like an UUID. I think we can add the prefix it's not going to do any harm and it can open some possibilities for extensions.

Reply:
I updated the SKUs so now they follow the convention `[3 chars from product type][3 chars from the product name]`

Reply:
It was `hex(3).upcase` :)